access-mgmt/README.adoc

```diff
@@ -0,0 +1,478 @@
+
+The following table lists the supported ACL permissions in {product}.
+
+NOTE: The resource identifier `Is` supports the wildcard character `*` to denote any occurrences of the specified resource. For example, `Topic` = `is *` means any topic in a Kafka instance.
+
+.Supported ACL permissions
+[cols="25%,25%,25%,25%"]
+|===
+h|Resource type
+h|Resource identifier
+h|Access type
+h|Operations
+
+|`Consumer group`
+
+(For consumer group access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Delete`
+* `Describe`
+
+|`Topic`
+
+(For access to a topic)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Write`
+* `Create`
+* `Delete`
+* `Alter`
+* `Alter configs`
+* `Describe`
+* `Describe configs`
+
+|`Transactional ID`
+
+(For producer access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Write`
+* `Describe`
+
+|`Kafka instance`
+
+(For access to Kafka instance permissions in ACLs)
+|None
+a|
+* `Allow`
+* `Deny`
+a|
+* `Alter`
+* `Describe`
+|===
+
+[role="_additional-resources"]
+.Additional resources
+https://kafka.apache.org/documentation/#security_authz_primitives[Authorization Primitives^] in Kafka documentation
+
+[id="proc-setting-permissions_{context}"]
+== Setting account permissions in a Kafka instance in {product}
+
+[role="_abstract"]
+In {product}, you can create Access Control Lists (ACLs) in your Kafka instances and set permissions for how other user accounts or service accounts can interact with an instance and its resources. You can manage access for only the Kafka instances that you create or for the instances that the owner has enabled you to access and alter.
+
+.Prerequisites
+* You've created a Kafka instance and the instance is in *Ready* state.
+* The user account or service account that you're setting permissions for has been created in the organization.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to set permissions for.
+. Click the *Permissions* tab to view the current ACL permissions for this instance.
++
+--
+By default, new Kafka instances contain the following permissions. These permissions allow all accounts in the organization to view the instance permissions and to view topics in the instance, but not to produce or consume messages.
+
+NOTE: The wildcard character `*` denotes any occurrences of the specified resource. For example, `Topic` = `is *` means any topic in a Kafka instance.
+
+.Default ACL permissions for new Kafka instances
+[cols="25%,25%,25%,25%"]
+|===
+h|Account
+h|Resource
+h|Access type
+h|Operation
+
+|`All accounts`
+|`Topic is *`
+
+(Any topic)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Consumer group is *`
+
+(Any consumer group)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Transactional ID is *`
+
+(Any producer)
+|`Allow`
+|`Describe`
```

Comment:
Missed this earlier, but I don't think we need to have describe on transactions enabled by default. There is no use for it in the UI that I am aware of.

Reply:
Sure, what should the default setting for transactions be then? And will the change to the UI be made for the milestone? This table simply describes what the default ACL is in the UI, so I have to follow whatever will actually be true in the UI for the milestone.

Reply:
I have no configuration at all for transactions in my notes. The ones out there currently (the compatibility rules) do still have "all" operations allowed for any transactional ID. We might need confirmation on this one.

Reply:
@MikeEdgar Okay, I'll wait to hear back from you on that. Feel free to ping others here if needed. Thanks.

Reply:
@sterobin - the defaults for transactional IDs can be removed.

Reply:
@MikeEdgar Awesome, fixed. Resolving. Thank you!